atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -727,7 +727,8 @@ public Map<ByteBuffer, List<ColumnOrSuperColumn>> apply(CassandraClient client)
 
                     @Override
                     public String toString() {
-                        return "multiget_multislice(" + host + ", " + tableRef + ", " + query.size() + " cells)";
+                        return "multiget_multislice(" + host.cassandraHostName() + ", " + tableRef + ", " + query.size()
+                                + " cells)";
```

Comment:
As far as I can tell, the `toString` method on `FunctionCheckedException` is called only in `CassandraRequestExceptionHandler`, where it is used as an unsafe arg in a log line.

I have a question: have you observed this on any other path? Can you confirm that we're only seeing this when we handle backoff due to an exception in the request?

And some thoughts:
* It seems a little odd that this is an unsafe arg. I know that table reference is unsafe and many include that, but it seems a bit daft that we cannot find out anything about the type of query just because of that. It may be an interesting idea to refactor this such that there is some useful information we can leverage safely (e.g. the hosts are safe, the size of the query, and the type of query), and then only use the unsafe args for the actually unsafe information.
* I wonder, realistically, how often we _ever_ look at this; I don't think I have ever requested unsafe logs just to view this information. We could perhaps move this to debug logging to help as well (whether that's just that extra arg, or the whole thing, I'm on the fence, since the safe args may have been used in the past).

My thoughts above are not a substitution to what you've done here - I think this is the right thing to do regardless.